README.Rmd

```diff
@@ -86,8 +86,8 @@ interface to Hector
 ## Contributing to Hector
 
 The Hector team welcomes and values community contributions, but please see 
-our [Contribution Guide](articles/ContributionsGuide.html) and note by  contributing 
-to this project, you agree to abide to our [Contributor Code of Conduct](CODE_OF_CONDUCT.html)
+our [Contribution Guide](vignettes/articles/ContributionsGuide.Rmd) and note by  contributing 
```

Comment:
Hmm so the reason why this was `articles/ContributionsGuide.html` was so that the linking worked on the homepage of the manual. This change seems to break that link though... what about switching to `https://jgcri.github.io/hector/articles/ContributionsGuide.html`?